src/lib/ai/services/ComparativeAnalyticsFactory.ts

```
@@ -0,0 +1,156 @@
+/**
+ * Factory for creating ComparativeAnalyticsService instances
+ *
+ * This module provides a factory function to create properly configured
+ * ComparativeAnalyticsService instances with the necessary dependencies.
+ */
+
+import {
+  ComparativeAnalyticsService,
+  ComparativeAnalyticsOptions,
+} from './ComparativeAnalyticsService'
```

ESLint

[error] Imports "ComparativeAnalyticsOptions" are only used as type.

(@typescript-eslint/consistent-type-imports)